pkgs/by-name/ox/oxipng/package.nix

```diff
@@ -21,8 +19,11 @@ rustPlatform.buildRustPackage rec {
   useFetchCargoVendor = true;
   cargoHash = "sha256-Z0otTCFwtGuSC1XBM3jcgGDFPZuMzQikZaYCnR+S6Us=";
 
-  # See https://github.com/shssoichiro/oxipng/blob/14b8b0e93a/.cargo/config.toml#L5
-  nativeCheckInputs = [ qemu ];
+  # don't require qemu for aarch64-linux tests
```

Comment:
```suggestion
```

Reply:
Not sure about this. This is intended to show that there are two reasons this file needs to be changed/removed

* qemu for aarch64-linux tests
* linker for aarch64-linux

Reply:
Oh I see, I thought the comment is regarding dropped line.